app/hooks/use-display-currency.ts

```diff
@@ -51,6 +53,13 @@ export const useDisplayCurrency = () => {
     [displayCurrency],
   )
 
+  const formatToUsd = useCallback((amount: number) => {
```

Comment:
What is the reason for including this in a hook? It has no dependencies and could a pure utility function.

Reply:
I put it there as something some query depends on that should be refactored. it could be living outside of it but if we put all the functions that will be in charged of either do a conversion or displaying a currency to a string, then it's easier to understand the requirement.